runtime/themes/nightfox.toml

```diff
@@ -0,0 +1,188 @@
+
+"ui.text" = { fg = "fg1" } # Default text color.
+"ui.text.focus" = { bg = "sel1", fg = "fg1" } # Selection highlight in buffer-picker or file-picker.
+"ui.text.info" = { fg = "fg2", bg = "sel0" } # Info popup contents (space mode menu).
+
+"ui.cursor" = { bg = "fg3", fg = "bg1" } # Fallback cursor colour, non-primary cursors when there are multiple (shift-c).
+"ui.cursor.primary" = { bg = "fg1", fg = "bg1" } # The primary cursor when there are multiple (shift-c).
+# "ui.cursor.insert" = { } # The cursor in insert mode (i).
+# "ui.cursor.select" = { } # The cursor in select mode (v).
+"ui.cursor.match" = { fg = "yellow", modifiers = ["bold"] } # The matching parentheses of that under the cursor.
+
+"ui.selection" = { bg = "bg3" } # All currently selected text.
+"ui.selection.primary" = { bg = "bg4" } # The primary selection when there are multiple.
+"ui.cursorline.primary" = { bg = "bg3" } # The line of the primary cursor (if cursorline is enabled)
+# "ui.cursorline.secondary" = { } #	The lines of any other cursors (if cursorline is enabled)
+# "ui.cursorcolumn.primary" = { } #	The column of the primary cursor (if cursorcolumn is enabled)
+# "ui.cursorcolumn.secondary" = { } #	The columns of any other cursors (if cursorcolumn is enabled)
+
+"ui.linenr" = { fg = "fg3" } # Line numbers.
+"ui.linenr.selected" = { fg = "yellow", modifiers = ["bold"] } # Current line number.
+
+# "ui.virtual" = { } # Namespace for additions to the editing area.
+"ui.virtual.ruler" = { bg = "bg3" } # Vertical rulers (colored columns in editing area).
+"ui.virtual.whitespace" = { fg = "bg3" } # Whitespace markers in editing area.
+"ui.virtual.indent-guide" = { fg = "black" } # Vertical indent width guides
+
+"ui.statusline" = { fg = "fg2", bg = "bg0" } # Status line.
+"ui.statusline.inactive" = { fg = "fg3", bg = "bg0" } # Status line in unfocused windows.
+"ui.statusline.normal" = { bg = "blue", fg = "bg0", modifiers = ["bold"] } # Statusline mode during normal mode (only if editor.color-modes is enabled)
+"ui.statusline.insert" = { bg = "green", fg = "bg0", modifiers = ["bold"] } # Statusline mode during insert mode (only if editor.color-modes is enabled)
+"ui.statusline.select" = { bg = "magenta", fg = "bg0", modifiers = ["bold"] } # Statusline mode during select mode (only if editor.color-modes is enabled)
+
+"ui.help" = { bg = "sel0", fg = "fg1" } # Description box for commands.
+
+"ui.menu" = { bg = "sel0", fg = "fg1" } # Code and command completion menus.
+"ui.menu.selected" = { bg = "fg3" } # Selected autocomplete item.
+"ui.menu.scroll"	= { fg = "fg3" } # fg sets thumb color, bg sets track color of scrollbar.
+
+"ui.popup" = { bg = "bg0", fg = "fg1" } # Documentation popups (space-k).
+"ui.popup.info" = { bg = "sel0", fg = "fg1" } # Info popups box (space mode menu).
+
+"markup.raw" = { fg = "magenta" } # Code block in Markdown.
+"markup.raw.inline" = { fg = "orange", modifiers = ["bold"] } # `Inline code block` in Markdown.
```

Comment:
There are a few other markup scopes that are very common to add. The full list is here https://github.com/helix-editor/helix/blob/0e23e4f882ec840bed0e2c1ef3723deb7a258801/book/src/themes.md?plain=1#L205

at least `markup.heading`, `markup.bold`, `markup.italic`, and `markup.link.url` and `markup.link.text` should be added or else markdown documents will not be highlighted very much